src/component-helpers.js

```diff
@@ -9,6 +9,10 @@ const componentsWithPendingUpdates = new WeakSet
 let syncUpdatesInProgressCounter = 0
 let syncDestructionsInProgressCounter = 0
 
+function validVirtualElement (virtualElement) {
+  return virtualElement != null && virtualElement !== false
```

Comment:
Is there any way to phrase this in positive terms, like looking for specific properties on the object?